app/controllers/admins/public_servants_controller.rb

```diff
@@ -121,12 +121,14 @@ def record_number_options
 
   def search_public_servants
     if params[:q].present?
-      @public_servants = @public_servants.where(name: params[:q][:name] ) unless params[:q][:name].blank?
+      @public_servants = @public_servants.
+        where(id: params[:q][:id]) unless params[:q][:id].blank?
       @public_servants = @public_servants.where(dependency: params[:q][:dependency]) unless params[:q][:dependency].blank?
       @public_servants = @public_servants.where(administrative_unit: params[:q][:administrative_unit] ) unless params[:q][:administrative_unit].blank?
       @public_servants = @public_servants.where(record_number:  params[:q][:record_number] ) unless params[:q][:record_number].blank?
 
-      @disabled_public_servants = @disabled_public_servants.where(name: params[:q][:name] ) unless params[:q][:name].blank?
+      @disabled_public_servants = @disabled_public_servants.
+        where(id: params[:q][:id] ) unless params[:q][:id].blank?
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Space inside parentheses detected.